apps/jclock/metadata.json

```diff
@@ -0,0 +1,16 @@
+{ "id":"jclock",
+  "name":"jclock",
+  "shortName":"jclock",
+  "icon":"app.png",
+  "version":"0.02",
+  "description":"Similar layout to Rebble clock, but much simpler features with switched time and the feature window. The time is on the right sie. This is my first Bagle app.",
```

Comment:
Typos: "sie" -> "side", "Bagle" -> "Bangle".

Reply:
Thank you for catching those typos.